src/KvkClient.php

```diff
@@ -39,6 +39,21 @@ public function getProfile(QueryInterface $profileQuery): KvkPaginatorInterface
         return $this->profileResponseFactory->fromProfileData($data);
     }
 
+    /**
+     * preform search query
+     * @param QueryInterface $profileQuery
+     * @return KvkPaginatorInterface
```

Comment:
my IDE (PhpStorm) requested to add this in the comments. Sure it is redunded but it has no harm to it

Reply:
> prone to be inconsistent/outdated
Don't you consider this harmful as it can lead to confusion to developers reading this code?

Anyway, the CS fixer will remove them now.